Danki2/Assets/Scripts/Effects/Dash.cs

```diff
@@ -19,7 +19,7 @@ float slowMultiplier
 
     protected override void FinishAction(Actor actor)
     {
-        actor.AddEffect(
+        actor.gameObject.GetComponent<EffectTracker>().AddEffect(
```

Comment:
Get the reference once, then use it repeatedly.

Reply:
I meant to delete this class actually as dash implemented outside effects